ci/azure-install-rust.yml

```diff
@@ -0,0 +1,14 @@
+parameters:
+  toolchain: 'stable'
+
```

Comment:
Maybe we should reuse this step in the build template?

Reply:
yeah i agree i was just trying to avoid to many moving parts- i'm happy to do it in another PR once we get the builds going :)